dap_prinz_green_jobs/utils/bert_vectorizer.py

```diff
@@ -1,7 +1,39 @@
 from sentence_transformers import SentenceTransformer
-import time
 from ojd_daps_skills import logger
+import numpy as np
+
+from dap_prinz_green_jobs.utils.processing import list_chunks
```

Comment:
i forgot about the processing utils library ! can also use toolz's `partition` method to split a list into equally-sized chunks.